src/isar/storage/utilities.py

```diff
@@ -0,0 +1,96 @@
+import json
+from pathlib import Path
+from typing import Any, Tuple
+from uuid import UUID
+
+from isar.models.mission_metadata.mission_metadata import MissionMetadata
+from isar.services.utilities.json_service import EnhancedJSONEncoder
+from robot_interface.models.inspection.inspection import Image, Inspection, ThermalImage
+
+
+def construct_local_paths(
+    inspection: Inspection, metadata: MissionMetadata
+) -> Tuple[Path, Path]:
+    inspection_type: str = get_inspection_type(inspection=inspection)
+
+    folder: Path = Path(str(metadata.mission_id))
+    filename: str = get_filename(
+        mission_id=metadata.mission_id,
+        inspection_type=inspection_type,
+        inspection_id=inspection.id,
+        file_type=inspection.metadata.file_type,
+    )
+    path: Path = folder.joinpath(filename)
+
+    metadata_filename: str = get_filename(
```

Comment:
Would it be cleaner to get the base file name and then append filetype and _meta.json for filename and metadata_filename respectively?

Reply:
Changed it so both of them use `filename` and added the file ending afterwards. Also changed so they have the exact same filename only differing file endings. So `_meta` is gone.

Reply:
Sounds good